docs/_pages/0005-blog.md

```diff
@@ -0,0 +1,27 @@
+---
+# /*
+# |    Protect your secrets, protect your sensitive data.
+# :    Explore VMware Secrets Manager docs at https://vsecm.com/
+# </
+# <>/  keep your secrets… secret
+# >/
+# <>/' Copyright 2023–present VMware, Inc.
+# >/'  SPDX-License-Identifier: BSD-2-Clause
+# */
+
+title: VSecM Blogs
```

Comment:
nitpick: Let’s use “Blog” (singular)

Blog is a place that has multiple blog posts.

“Blogs” implies that we have different kinds of blogs (a design blog, a tech blog, a business blog etc) — which is not the case — Well, at least, not… yet :D .

Reply:
Make sense. 👍